src/Microsoft.Extensions.Logging.Testing/Microsoft.Extensions.Logging.Testing.csproj

```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="xunit.extensibility.core" Version="$(XunitVersion)" />
```

Comment:
Can this be xunit.abstractions? We only need [ITestOutputHelper](https://github.com/xunit/abstractions.xunit/blob/master/src/xunit.abstractions/TestCases/ITestOutputHelper.cs), right?